misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathAssertionsSpec.kt

```diff
@@ -115,25 +106,20 @@ abstract class PathAssertionsSpec(
                 }
             }
         }
-    }
 
-    describeFun(startsNotWith.name) {
-        val startsNotWithFun = startsNotWith.lambda
         context("does not start with") {
-            it("does not throw") {
+            it("${startsNotWith.name} does not throw") {
                 expect(Paths.get("/some/path/for/test"))
                     .startsNotWithFun(Paths.get("/other/path/"))
             }
-            it("does not match partials") {
+
+            it("${startsNotWith.name} does not match partials") {
                 expect(Paths.get("/some/path/for/test"))
                     .startsNotWithFun(Paths.get("/some/pa"))
             }
-        }
-
-        val expectedMessageIfStartsWith = "${DescriptionPathAssertion.STARTS_NOT_WITH.getDefault()}:"
 
-        context("starts with") {
-            it("throws an AssertionError") {
+            it("${startsNotWith.name} throws an AssertionError") {
```

Comment:
same here, I guess you don't require two contexts, one path is enough, you could even inlcude endsWith/endsNotWith in this context

Reply:
Thanks! I used the same context for `startsWith`/`startsNotWith`, but I had a different `describeFun` containing just one `context` for `endsWith`/`endsNotWith` like `...integration/CharSequenceAssertionsSpec`

I hope that's fine?